### PR TITLE
Route Server in BGP peering with Quagga

### PR DESCRIPTION
**Pull request by fabferri**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by fabferri** [comment]
Hello,
can you please let me know if all is OK to merge the pull request?
Thanks

File: 101-route-server-quagga/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+        "scriptFolder": "scripts",
+        "customScriptFilePath": "[uri(parameters('_artifactsLocation'), concat(variables('scriptFolder'), '/', 'quaggadeploy.sh', parameters('_artifactsLocationSasToken')))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "name": "[variables('nsgName')]",
+            "apiVersion": "2020-06-01",
+            "location": "[variables('location')]",
+            "comments": "network security group",
```
Comment (bmoore-msft) [inline 101-route-server-quagga/azuredeploy.json:237]: See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

File: 101-route-server-quagga/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+            }
+        },
+        "vmPrivIP": {
+            "type": "string",
+            "defaultValue": "10.1.4.10",
+            "metadata": {
+                "description": "private IP of the VM. This IP has to match with the IP specified in the quaggadeploy.sh bash script"
+            }
+        },
+        "publisher": {
```
Comment (bmoore-msft) [inline 101-route-server-quagga/azuredeploy.json:149]: set allowedValues on the vm image properties

File: 101-route-server-quagga/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,467 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```
Comment (bmoore-msft) [inline 101-route-server-quagga/azuredeploy.json:5]: add a defaultValue [resourceGroup().location] and remove the param value from the param file

File: 101-route-server-quagga/azuredeploy.parameters.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location":{
+            "value": "eastus"
```
Comment (bmoore-msft) [inline 101-route-server-quagga/azuredeploy.parameters.json:6]: remove this and use a defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-route-server-quagga/azuredeploy.parameters.json (hunk 4 above)
Comment (fabferri) [reply 101-route-server-quagga/azuredeploy.parameters.json]: done

File: 101-route-server-quagga/azuredeploy.json (hunk 3 above)
Comment (fabferri) [reply 101-route-server-quagga/azuredeploy.json:5]: done

File: 101-route-server-quagga/azuredeploy.json (hunk 2 above)
Comment (fabferri) [reply 101-route-server-quagga/azuredeploy.json]: added the variable:
"imageReference": {
            "publisher": "Canonical",
            "offer": "0001-com-ubuntu-server-focal",
            "sku": "20_04-lts",
            "version": "latest"
        },

File: 101-route-server-quagga/azuredeploy.json (hunk 1 above)
Comment (fabferri) [reply 101-route-server-quagga/azuredeploy.json]: done

**Comment by fabferri** [comment]
All the changes applied and checks OK. Please check it out.

**Pull request merged**